src/store/initial-state.js

```diff
@@ -91,13 +91,26 @@ export const prepareNonPipelineState = (props) => {
 
 /**
  * Configure the redux store's initial state, by merging default values
- * with normalised pipeline data and localStorage
+ * with normalised pipeline data and localStorage.
+ * If parameters flag is set to true, then disable parameters on initial load
  * @param {object} props App component props
  * @return {object} Initial state
  */
-const getInitialState = (props = {}) => ({
-  ...prepareNonPipelineState(props),
-  ...preparePipelineState(props.data, props.data !== 'json'),
-});
+const getInitialState = (props = {}) => {
+  const nonPipelineState = prepareNonPipelineState(props);
+  const storedState = loadState();
+  if (nonPipelineState.flags.newparams) {
+    saveState({
```

Comment:
(optional) I would move this variable into the `if` block:
```suggestion
  if (nonPipelineState.flags.newparams) {
    const storedState = loadState();
    saveState({
```